system-tests/src/test/java/org/eclipse/edc/samples/transfer/streaming/Streaming01httpToHttpTest.java

```diff
@@ -97,14 +96,24 @@ void streamData() throws IOException {
         PROVIDER.createPolicyDefinition(getFileContentFromRelativePath(SAMPLE_FOLDER + "/policy-definition.json"));
         PROVIDER.createContractDefinition(getFileContentFromRelativePath(SAMPLE_FOLDER + "/contract-definition.json"));
 
-        var destination = Json.createObjectBuilder()
-                .add("type", "HttpData")
-                .add("baseUrl", "http://localhost:" + httpReceiverPort)
-                .build();
-        var transferProcessId = CONSUMER.requestAssetFrom("stream-asset", PROVIDER)
-                .withDestination(destination)
-                .withTransferType("HttpData-PUSH")
-                .execute();
+        var catalogDatasetId = CONSUMER.fetchDatasetFromCatalog(getFileContentFromRelativePath(SAMPLE_FOLDER + "/get-dataset.json"));
+        var negotiateContractBody = getFileContentFromRelativePath(SAMPLE_FOLDER + "/negotiate-contract.json")
+                                        .replace("{{offerId}}", catalogDatasetId);
+        //assertThat(catalogDatasetId).isEqualTo("test", catalogDatasetId);
```

Comment:
commented out line should be removed

Reply:
Oh, good catch.
Had been too long yesterday, obviously.
That was my way to find out how to fix the issue that jsonPath rejects colons if not escaped.
A bit tricky when it is the first time to use it at all ;-)

I have amended it to the last commit to not have too many.